src/System.Private.CoreLib/src/System/Reflection/RtFieldInfo.cs

```diff
@@ -109,11 +109,11 @@ internal void CheckConsistency(object target)
             }
         }
 
-        internal override bool CacheEquals(object o)
+        internal override bool CacheEquals(object? o)
         {
-            RtFieldInfo m = o as RtFieldInfo;
+            RtFieldInfo? m = o as RtFieldInfo;
 
-            if ((object)m == null)
+            if (m is null)
```

Comment:
Ditto.  Same anywhere else a similar change is made.

Reply:
this one seems correct per your previous comment. just validated all similar changes and they seem fine